repos/fdbt-site/src/pages/products/productDetails.tsx

```diff
@@ -305,7 +306,24 @@ const createProductDetails = async (
     }
 
     if ('zoneName' in ticket) {
-        productDetailsElements.push({ id: 'zone', name: 'Zone', content: [ticket.zoneName] });
+        productDetailsElements.push({
+            id: 'zone',
+            name: 'Zone',
+            content: [ticket.zoneName],
+            editLink: '/csvZoneUpload',
+        });
+    }
+
+    if (isGeoZoneTicket(ticket)) {
```

Comment:
Can we not use the same if block above? zoneName in ticket means its a geoZone I think

Reply:
causes a type script error if I just use zoneName so i will put them all in the same block with the isGeozoneTicket check